docker/celerybeat/Dockerfile

```diff
@@ -10,6 +10,7 @@ WORKDIR /usr/src/app
 COPY ./backend/requirements.txt /usr/src/app/.
 
 RUN pip3 install -r requirements.txt
+RUN pip3 install watchdog argh
```

Comment:
Should this be before `RUN pip3 install -r requirements.txt`, maybe even before `RUN pip3 install -r requirements.txt`  because we expect this layer to change less often so we want to keep this as a cached layer and not rebuild it every time we change the requirements.

Reply:
Yes, but I don't know if we want to add this as a requirement or not.

Reply:
I was not suggesting to put it in the requirements.txt just adding this command to the line 9 of the dockerfile instead of the line 13 to keep it cached for docker builds as it will probably change less often.

Reply:
Yes, it's my suggestion, if we put in in the requirements.txt we don't need to swap lines but only to remove one :)

Reply:
As all images we build depend on this I think we should put it in the requirements.txt, I think it's better to manage all the python dependencies in the same place.
Adding it to the Dockerfile we make this a dependency of the images we publish even if it is not really required.